diff --git a/server/pom.xml b/server/pom.xml
index e9f5bd6..ea57195 100644
--- a/server/pom.xml
+++ b/server/pom.xml
@@ -11,7 +11,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Quarkus configuration -->
-        <quarkus.platform.version>3.0.3.Final</quarkus.platform.version>
+        <quarkus.platform.version>2.16.7.Final</quarkus.platform.version>
     </properties>
 
     <modelVersion>4.0.0</modelVersion>
diff --git a/server/server-common/src/main/java/com/github/jdussouillez/server/service/PdfService.java b/server/server-common/src/main/java/com/github/jdussouillez/server/service/PdfService.java
index fb7fce9..01d5efc 100644
--- a/server/server-common/src/main/java/com/github/jdussouillez/server/service/PdfService.java
+++ b/server/server-common/src/main/java/com/github/jdussouillez/server/service/PdfService.java
@@ -1,7 +1,7 @@
 package com.github.jdussouillez.server.service;
 
 import io.smallrye.mutiny.Uni;
-import jakarta.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.ApplicationScoped;
 import java.nio.charset.StandardCharsets;
 import java.time.Duration;
 
diff --git a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/EnableGrpcErrorManagement.java b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/EnableGrpcErrorManagement.java
index 2d3f0d3..40285cb 100644
--- a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/EnableGrpcErrorManagement.java
+++ b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/EnableGrpcErrorManagement.java
@@ -1,6 +1,6 @@
 package com.github.jdussouillez.server.rpc.interceptor;
 
-import jakarta.interceptor.InterceptorBinding;
+import javax.interceptor.InterceptorBinding;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
diff --git a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/GrpcErrorManager.java b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/GrpcErrorManager.java
index 96feb96..9be5562 100644
--- a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/GrpcErrorManager.java
+++ b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/interceptor/GrpcErrorManager.java
@@ -6,10 +6,10 @@ import io.grpc.Status;
 import io.grpc.StatusException;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
-import jakarta.annotation.Priority;
-import jakarta.interceptor.AroundInvoke;
-import jakarta.interceptor.Interceptor;
-import jakarta.interceptor.InvocationContext;
+import javax.annotation.Priority;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
 
 @Interceptor
 @Priority(2000)
diff --git a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/FileChunkMapper.java b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/FileChunkMapper.java
index bd6000e..cec23a8 100644
--- a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/FileChunkMapper.java
+++ b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/FileChunkMapper.java
@@ -3,7 +3,7 @@ package com.github.jdussouillez.server.rpc.mapper;
 import com.github.jdussouillez.api.grpc.FileChunk;
 import com.github.jdussouillez.server.utils.ByteUtils;
 import com.google.protobuf.ByteString;
-import jakarta.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.ApplicationScoped;
 import java.util.Collection;
 
 @ApplicationScoped
diff --git a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/PdfRequestMapper.java b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/PdfRequestMapper.java
index fbf6c0a..7598649 100644
--- a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/PdfRequestMapper.java
+++ b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/mapper/PdfRequestMapper.java
@@ -1,7 +1,7 @@
 package com.github.jdussouillez.server.rpc.mapper;
 
 import com.github.jdussouillez.api.grpc.PdfRequest;
-import jakarta.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.ApplicationScoped;
 import java.util.Collection;
 import java.util.stream.Collectors;
 
diff --git a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/service/PdfApiService.java b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/service/PdfApiService.java
index df5a41e..520fb96 100644
--- a/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/service/PdfApiService.java
+++ b/server/server-rpc/src/main/java/com/github/jdussouillez/server/rpc/service/PdfApiService.java
@@ -10,7 +10,7 @@ import com.github.jdussouillez.server.service.PdfService;
 import io.quarkus.grpc.GrpcService;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.infrastructure.Infrastructure;
-import jakarta.inject.Inject;
+import javax.inject.Inject;
 
 @GrpcService
 @EnableGrpcErrorManagement
